Expand the Getting started doc (#612)
memgraph/documentation: pages/getting-started.mdx

Get Memgraph up and running on your data by following these installation steps. 

<Callout> 

> This installation script uses a `docker-compose.yml` file that we have prepared.

Reviewer: ```suggestion
This installation script uses a pre-prepared `docker-compose.yml` file. 
```

You can download it from
[https://github.com/memgraph/memgraph-platform](https://github.com/memgraph/memgraph-platform).

> Feel free to modify it according to your needs, and then run it using [Docker

Reviewer: Maybe add link to **[configuration settings](https://memgraph.com/docs/configuration/configuration-settings)**

Lab](/data-visualization) and a complete set of algorithms within a
[MAGE](/advanced-algorithms) library.

### Install Memgraph Platform

Ensure [Docker](https://docs.docker.com/engine/install/) is running in the
background. Depending on your operating system, execute the appropriate command
in the console.

#### Linux and macOS

<Steps>

{<h3>Open a terminal window on your machine.</h3>}

{<h3>Execute the following command</h3>}

```bash
curl https://install.memgraph.com | sh
```

This command downloads the installation script from Memgraph's website and runs
it, installing the latest version of Memgraph on your Linux or macOS system.
</Steps>

#### Windows

<Steps>

> {<h3>Open a PowerShell window on your machine.</h3>} 

Reviewer: ```suggestion
{<h3>Open a PowerShell terminal on your machine.</h3>} 
```

{<h3>Execute the following command:.</h3>}

```powershell
iwr https://windows.memgraph.com | iex
```

This command uses PowerShell's `Invoke-WebRequest` (`iwr`) cmdlet to fetch the
installation script from Memgraph's website and executes it, installing the
latest version of Memgraph on your Windows system.

</Steps>

<Callout> 

To gracefully stop and remove all running Memgraph containers managed by Docker

> Compose, use the following command from the directory containing your
> `docker-compose.yml` file:

Reviewer: ```suggestion
Compose, place yourself in the terminal in the directory containing your `docker-compose.yml` file and run the following command:
```
